src/components/VersionSwitcher/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { View } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import flatDirectory from 'src/directory/flatDirectory.json';
+
+
+const findRoute = (platform, isPrev) => {
```

Comment:
This looks for the corresponding page and if so the link will point to that page otherwise just the base page